README.md

```diff
@@ -1,15 +1,17 @@
 # webp
 
-Asset transformer for converting images into WebP files.
-
-## Install cwebp
-### Option 1: Download precompiled binaries from Google
+[![webp pub.dev badge][pub-badge]][pub-badge-link]
+[![][build-badge]][build-badge-link]
 
-Follow the instructions at [https://developers.google.com/speed/webp/docs/precompiled](https://developers.google.com/speed/webp/docs/precompiled) to download and install the precompiled binaries for cwebp.
+Asset transformer for converting images into WebP files.
 
-### Option 2: Install using Homebrew (macOS)
+## Supported architectures
```

Comment:
Let's move it to the end (above Learn more) of the README. These architectures will fit 99% of the users. You can also add a sentence of explanation why do we _support architectures_ (because we provide binaries ourselves)